### ISSUE TITLE
sandmark requires the system package libdw-dev on Ubuntu

### ISSUE THREAD

**Issue by UnixJunkie** [ocaml-bench/sandmark#101]
Since there is no opam file, its installation is not automatic.

Maybe, there should be an opam file.

**Comment by kayceesrk** [comment]
The easiest way now would be to add a `Make` rule called `depend` which installs `libdw-dev`. I'm not sure if an OPAM file is the right way to install system dependencies. @shakthimaan would you be able to do this?

**Comment by shakthimaan** [comment]
Sure, but, the package names are specific to .deb based distributions. Actually, you require both `libgmp-dev` and `libdw-dev` to be installed on Ubuntu, and this is mentioned in the "pre-requisities" section in the README file.

**Comment by UnixJunkie** [comment]
opam is the right way to install system dependencies.
E.g. in an opam file:
```
depexts: [
  ["gnuplot-x11"] {os-family = "debian"}
  ["gnuplot"] {os-distribution = "centos"}
  ["gnuplot"] {os-distribution = "fedora"}
  ["gnuplot"] {os = "macos" & os-distribution = "homebrew"}
  ["gnuplot"] {os-distribution = "alpine"}
  ["gnuplot"] {os-family = "suse"}
]
```
Allows to create package installs depending on the Linux distro.
Supporting Debian and Ubuntu would be the very minimum.
Usually, in opam, conf-* packages are full of such statements.

**Comment by shakthimaan** [comment]
I shall add an opam file for sandmark shortly. If you already have one that we can use, please feel free to create a PR. Thanks!

**Comment by UnixJunkie** [comment]
I don't have one, sorry.

**Comment by kayceesrk** [comment]
We also need to ensure python3 and pip3 is installed. And also intervaltree library. 

```bash
$ sudo apt install python3 python3-pip
$ pip3 install intervaltree
```

**Comment by kayceesrk** [comment]
Addressed by #103 and #104.
